phoenix-core/src/main/java/org/apache/phoenix/coprocessor/PhoenixAccessController.java

```diff
@@ -464,12 +464,14 @@ public void preIndexUpdate(ObserverContext<PhoenixMetaDataControllerEnvironment>
          return userPermissions;
        }
 
+     //FIXME This seems to have no effect at all
      private void getUserDefinedPermissions(final TableName tableName,
              final List<UserPermission> userPermissions) throws IOException {
           User.runAsLoginUser(new PrivilegedExceptionAction<List<UserPermission>>() {
               @Override
               public List<UserPermission> run() throws Exception {
-                  final List<UserPermission> userPermissions = new ArrayList<UserPermission>();
+                 //FIXME We are masking the parameter list that we are supposed to add to
```

Comment:
A todo you left for yourself? :)

Reply:
Yes, I don't want fix unrelated bugs in this commit, to make testing easier.
Also, writing a test case for this promises to be quite hairy, which is another reason not to tackle this right now.
I'll make a separate ticket for this as well.

Reply:
Ah, I see now. I just read it too fast. That sounds fine!

Reply:
https://issues.apache.org/jira/browse/PHOENIX-5700